qemu/tests/blockdev_commit_general_operation.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from provider import job_utils
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitGeneralOperation(BlockDevCommitTest):
+
+    def commit_op(self, cmd, args=None):
+        self.main_vm.monitor.cmd(cmd, args)
+        job_status = job_utils.query_block_jobs(self.main_vm)
```

Comment:
All status/events can be found in the qmp cmd log file, fyi

Reply:
Yes, just used to view the result in debug.log after operation.Thanks.